common/views/components/BaseHeader/ExhibitionHeader.js

```diff
@@ -4,30 +4,23 @@ import {Fragment} from 'react';
 import HeaderText from '../HeaderText/HeaderText';
 import WobblyEdge from '../WobblyEdge/WobblyEdge';
 import FreeSticker from '../FreeSticker/FreeSticker';
-import type {Link} from '../../../model/link';
 import {BasePageColumn} from '../BasePage/BasePage';
 import {classNames, spacing} from '../../../utils/classnames';
 import type {Node} from 'react';
 import type {FeaturedMedia as FeaturedMediaType} from '../BaseHeader/BaseHeader';
 
 type Props = {|
   title: string,
-  TagBar: ?Node,
   DateInfo: ?Node,
   InfoBar: ?Node,
   FeaturedMedia: ?FeaturedMediaType,
-  topLink: ?Link,
-  Description: ?Node
 |}
 
-const ImageLeadHeader = ({
+const ExhibitionHeader = ({
```

Comment:
We have no logic yet about when to group components, do we have a reason why we have here? It makes it hard to know where to import things from when not only 1 layer deep (guilty of having done this, and was thinking of going back to tidy it up)?

Reply:
Agreed – let's go flat.